main.tf

```diff
@@ -65,167 +72,4 @@ locals {
-    from_port   = local.ingress_controller_node_ports.http
-    to_port     = local.ingress_controller_node_ports.http
-    protocol    = "tcp"
-    cidr_blocks = [
-      "0.0.0.0/0",
-    ]
-  }
-}
-
-resource "aws_security_group" "worker_https_ingress" {
-  name_prefix = "${var.cluster_name}-ingress-https-"
-  description = "Allows HTTPS access from anywhere to the ingress NodePorts"
-  vpc_id      = data.aws_vpc.selected.id
-
-  ingress {
-    description = "HTTPS from NLB"
-    from_port   = local.ingress_controller_node_ports.https
-    to_port     = local.ingress_controller_node_ports.https
-    protocol    = "tcp"
-    cidr_blocks = [
-      "0.0.0.0/0",
-    ]
-  }
-}
-
-module "eks" {
-  source = "github.com/terraform-aws-modules/terraform-aws-eks?ref=v12.0.0"
-
-  cluster_name    = var.cluster_name
-  cluster_version = var.cluster_version
-
-  enable_irsa = true
-  vpc_id      = data.aws_vpc.selected.id
-  subnets     = var.worker_subnet_ids
-
-  worker_groups_launch_template = [
-    {
-      platform                = "linux"
-      asg_max_size            = var.linux_workers_count
-      asg_min_size            = var.linux_workers_count
-      asg_desired_capacity    = var.linux_workers_count
-      override_instance_types = ["t3.large"]
-      spot_instance_pools     = 1
-
-      target_group_arns = module.lb.target_group_arns
-
-      additional_security_group_ids = [
-        aws_security_group.worker_http_ingress.id,
-        aws_security_group.worker_https_ingress.id,
-      ]
-
-      # These indicate that this ASG will participate in auto-scaling
-      # through the cluster-autoscaler.
-      tags = [for k, v in local.cluster_autoscaler.asg_tags : {
-        key                 = k
-        value               = v
-        propagate_at_launch = false
-      }]
-    },
-
-    {
-      name                    = "windows-worker-group"
-      platform                = "windows"
-      asg_max_size            = var.windows_workers_count
-      asg_min_size            = var.windows_workers_count
-      asg_desired_capacity    = var.windows_workers_count
-      override_instance_types = ["m5.large"]
-      spot_instance_pools     = 1
-
-      # These indicate that this ASG will participate in auto-scaling
-      # through the cluster-autoscaler.
-      tags = [for k, v in local.cluster_autoscaler.asg_tags : {
-        key                 = k
-        value               = v
-        propagate_at_launch = false
-      }]
-    }
-  ]
-
-  tags = local.combined_tags
-}
-
-data "http" "aws_vpc_cni" {
-  url = "https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/master/config/v${local.aws_vpc_cni_version}/aws-k8s-cni.yaml"
-}
-
-resource "null_resource" "aws_vpc_cni" {
-  provisioner "local-exec" {
-    command = <<EOF
-cat <<MOF | kubectl apply -f -
-${data.http.aws_vpc_cni.body}
-MOF
-EOF
-
-    environment = {
-      KUBECONFIG = "${module.eks.kubeconfig_filename}"
-    }
-  }
-}
-
-resource "null_resource" "windows_support" {
```

Comment:
I'm sad to see this go. We need this 👀

Reply:
I don't want it to go away, i think it should be moved to a module/submodule